### PR TITLE
4.0-7.10 Backend organization

### PR DESCRIPTION
**Pull request by Desvelao**
Hi team, this PR:

- Removed not used files (`.old.js`)
- Renamed `.js` files to `.ts`
- Renamed `/util` to `/common`
- Moved `/server/reporting` to `/server/lib/reporting`
- Moved `/server/jobs/queue` to `/server/start/queue`
- Moved `/server/logger` to `/server/lib/logger`
- Moved files in `/util` to `/server/lib`
- Adapted imports in backend and frontend
- Created `index` files for each backend job
- Removed endpoints:
  - `/api/pci/{requirement}`
  - `/api/gdpr/{requirement}`
  - `/api/nist/{requirement}`
  - `/api/tsc/{requirement}`
  - `/api/hipaa/{requirement}`
  - `/api/monitoring`
  - `/api/agents`
  - `/elastic/index-patterns`
  - `/reports`
- Renamed endpoint `/elastic/esAlerts` to `/elastic/alerts`. Adapted frontend
- Removed files:
  - `/server/lib/elastic-wrapper.ts`
  - `/server/lib/api-helper.js`
  - `/server/lib/get-path.ts`
  - `/server/lib/reporting/tab-description`

### CODE REVIEW

**Comment by Desvelao** [comment]
Fixed relative path imports of fonts and logo to create reports.

File: public/components/common/modules/overview-current-section.tsx [hunk 1]
```diff
@@ -80,10 +80,10 @@ class WzCurrentOverviewSection extends Component {
...
       <span>
-      {/*this.props.currentTab && TabDescription[this.props.currentTab] && TabDescription[this.props.currentTab].title && (
+      {/*this.props.currentTab && WAZUH_MODULES[this.props.currentTab] && WAZUH_MODULES[this.props.currentTab].title && (
       <EuiTitle size='s'>
         <h2>
-          {TabDescription[this.props.currentTab].title}
+          {WAZUH_MODULES[this.props.currentTab].title}
        </h2>
       </EuiTitle>)*/}
         </span>
```
Comment (gabiwassan) [inline public/components/common/modules/overview-current-section.tsx:89]: This component is deprecated?
`overview-current-section.tsx`

**Review by frankeros — APPROVED** [review]
LGTM! :tada:

**Review by frankeros — APPROVED** [review]
LGTM! :tada:

**Review by gabiwassan — APPROVED** [review]
Great job!

**Pull request merged**